app/assets/stylesheets/static_pages.scss

```
@@ -44,3 +44,31 @@ textarea {
     color: $whiteColor;
     text-decoration: none;
 }
+img {
+    display: none;
+}
+
+/* footer */
+footer {
```

line 52 -> 71 dùng nested em sửa lại như sau
```suggestion
footer {
    margin-top: 45px;
    padding-top: 5px;
    border-top: 1px solid $whiteColor;
    color: $greyColor;
    
    a {
        color: $blackColor;
    }
    ...
    small {
        float: left;
    }
    ...
}
```